sunkit_instruments/suvi/suvi.py

```diff
@@ -178,6 +173,12 @@ def get_response(request, spacecraft=16, ccd_temperature=-60.0, exposure_type="l
             f"Valid spacecraft are: {VALID_SPACECRAFT}"
         )
 
+    # Ensure ccd_temperature is a Quantity in degrees Celsius
+    if not isinstance(ccd_temperature, u.Quantity):
+        ccd_temperature = ccd_temperature * u.deg_C
+    else:
+        ccd_temperature = ccd_temperature.to(u.deg_C)
```

Comment:
If you use https://docs.astropy.org/en/stable/api/astropy.units.quantity_input.html#astropy.units.quantity_input none of this is needed.